### PR TITLE
Remix flow

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
A couple things:

1. Attribution in the modal is TODO. I don't have the original author's username coming through from the service. I'm going to file a ticket for that...

2. You have to drag to refresh on the Make view to see your remixes show up. Is this OK? Should we force that view to update every time it's loaded?

**Comment by gvn** [comment]
Related: #1569

File: www_src/pages/project/project.jsx [hunk 1]
```diff
@@ -199,6 +199,74 @@ var Project = React.createClass({
...
+        }
+
+        var projectID = data.project.id;
+        var projectTitle = data.project.title;
+
+        // Get author's username
+        api({
+          method: 'GET',
+          uri: `/users/${this.state.params.user}/projects/${this.state.params.project}`
+        }, (err, moreData) => {
```
Comment (thisandagain) [inline www_src/pages/project/project.jsx]: Handle the error

Comment (gvn) [inline www_src/pages/project/project.jsx]: Good call. Fixed!

**Comment by thisandagain** [comment]
Awesome how clean / tiny this is. R+

**Pull request merged**